cal/src/components/Calendar/Week.jsx

```diff
@@ -1,44 +0,0 @@
-import React from 'react'
```

Comment:
@dahyunjlee Yep, got moved into its own folder called `Week` in the `Calendar` folder and is now called `index.jsx`